routes/deductions/deductions.spec.js

```diff
@@ -97,6 +97,7 @@ describe('Test /deductions responses', () => {
         url: '/trillium/studentResidence',
         key: 'trilliumStudentResidence',
         yesRedir: '/success',
+        noAmountPage: true,
```

Comment:
This key is kind of the same as the other one. ie, if there is a "yesRedir", then there is no amount page.

Let's just use the existence of the other key to do this check.